config/web-app/istio.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
```suggestion
apiVersion: networking.istio.io/v1beta1
```

Reply:
Good catch, thanks!

Reply:
I also added an extra commit since we will also need to use `v1beta1` in the patch selector, inside the `overlays/kubeflow/kustomization.yaml` file, as well.